cli/worker.rs

```diff
@@ -371,11 +371,12 @@ mod tests {
     })
   }
 
-  /*
   #[test]
   fn removed_from_resource_table_on_close() {
     tokio_util::init(|| {
-      let worker = create_test_worker();
+      let mut worker = create_test_worker();
+      js_check(worker.execute("onmessage = () => close();"));
```

Comment:
Note: This (or one of these) errors even though the tests pass:

```
onmessage = () => close();
                  ^
Uncaught ReferenceError: close is not defined
    at onmessage (<anonymous>:1:19)
```

Reply:
thanks - fixed.